extensions/ql-vscode/src/query-history-info.ts

```diff
@@ -17,3 +17,16 @@ export function getRawQueryName(item: QueryHistoryInfo): string {
       assertNever(item);
   }
 }
+
+export function getQueryId(item: QueryHistoryInfo): string {
```

Comment:
Can we name this `getQueryHistoryItemId` instead? I think `queryId` isn't quite correct - we're just looking for something that identifies and separates between items in the query history.